pkg/sizing/algorithm_test.go

```diff
@@ -0,0 +1,35 @@
+package sizing
+
+import (
+	"testing"
+	"testing/quick"
+
+	"github.com/stretchr/testify/require"
+)
+
+func Test_AlgorithTest_Algorith(t *testing.T) {
+	f := func(ingest int) bool {
+		if ingest < 0 {
+			ingest = -ingest
+		}
+		replicas := true
+		for _, cloud := range NodeTypesByProvider {
+			for _, node := range cloud {
+				size := calculateClusterSize(node, ingest, Basic)
+				replicas = size.totalNodes > 0.0 && size.totalReadReplicas > 0.0 && size.totalWriteReplicas > 0.0
+				require.Truef(t, replicas, "Cluster size was empty: ingest=%d cluster=%v node=%v", ingest, size, node)
+				require.InDelta(t, size.totalReadReplicas, size.totalWriteReplicas, 5.0, "Replicas have different sizes: ingest=%d node=%s", ingest, node.name)
```

Comment:
This assumes that the replicas should be about equal. For very large input they are not. Interestingly the algorithm requires  >3000 replicas if the ingest is 1 Petabyte.